From 0328d1d60b8938f83aa39cbb1209a1cea3577cc6 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 27 Jun 2024 11:27:20 -0700
Subject: [PATCH] Docs/mm/index: move allocation profiling document to unsorted
 documents chapter

The memory allocation profiling document was added to the bottom of the
new outline not because people concluded it is the best place for the
document after deep discussions or following a well defined guideline.
Rather than that, it was added there just because it is not legacy
document but not clear where it should be added.  Move it to unsorted
documents chapter.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/index.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/mm/index.rst b/Documentation/mm/index.rst
index 15614df2b989..0be1c7503a01 100644
--- a/Documentation/mm/index.rst
+++ b/Documentation/mm/index.rst
@@ -23,7 +23,6 @@ see the :doc:`admin guide <../admin-guide/mm/index>`.
    page_cache
    shmfs
    oom
-   allocation-profiling
 
 Unsorted Documentation
 ======================
@@ -38,6 +37,7 @@ documentation, or deleted if it has served its purpose.
    :maxdepth: 1
 
    active_mm
+   allocation-profiling
    arch_pgtable_helpers
    balance
    damon/index
-- 
2.39.2

